frameworks/blacksheep_app.py

```diff
@@ -149,74 +228,91 @@ def sync_response_cookies() -> Response:
+    content = await anyio.Path(test_data.FILE_10K).read_bytes()
+    return file(content, file_name="response_file", content_type="application/x-binary")
+
+
+@app.router.get("/async-file-response-100K")
+async def async_file_response_100k() -> Response:
+    content = await anyio.Path(test_data.FILE_100K).read_bytes()
+    return file(content, file_name="response_file", content_type="application/x-binary")
+
+
+@app.router.get("/async-file-response-500K")
+async def async_file_response_500k() -> Response:
+    content = await anyio.Path(test_data.FILE_500K).read_bytes()
     return file(content, file_name="response_file", content_type="application/x-binary")
 
 
 @app.router.get("/async-file-response-1M")
 async def async_file_response_1m() -> Response:
-    content = await anyio.Path(test_data.RESPONSE_FILE_1M).read_bytes()
+    content = await anyio.Path(test_data.FILE_1M).read_bytes()
     return file(content, file_name="response_file", content_type="application/x-binary")
 
 
-@app.router.get("/sync-file-response-100B")
-def sync_file_response_100b() -> Response:
-    return file(
-        test_data.RESPONSE_FILE_100B.read_bytes(), file_name="response_file", content_type="application/x-binary"
-    )
+@app.router.get("/async-file-response-5M")
+async def async_file_response_5m() -> Response:
+    content = await anyio.Path(test_data.FILE_5M).read_bytes()
+    return file(content, file_name="response_file", content_type="application/x-binary")
+
 
+# files sync
 
-@app.router.get("/sync-file-response-50K")
-def sync_file_response_50k() -> Response:
-    return file(
-        test_data.RESPONSE_FILE_50K.read_bytes(), file_name="response_file", content_type="application/x-binary"
-    )
+
+@app.router.get("/sync-file-response-100B")
+def sync_file_response_100b() -> Response:
+    content = test_data.FILE_100B.read_bytes()
+    return file(content, file_name="response_file", content_type="application/x-binary")
 
 
 @app.router.get("/sync-file-response-1K")
 def sync_file_response_1k() -> Response:
-    return file(test_data.RESPONSE_FILE_1K.read_bytes(), file_name="response_file", content_type="application/x-binary")
+    content = test_data.FILE_1K.read_bytes()
+    return file(content, file_name="response_file", content_type="application/x-binary")
+
+
+@app.router.get("/sync-file-response-10K")
+def sync_file_response_10k() -> Response:
+    content = test_data.FILE_10K.read_bytes()
+    return file(content, file_name="response_file", content_type="application/x-binary")
+
+
+@app.router.get("/sync-file-response-100K")
+def sync_file_response_100k() -> Response:
+    content = test_data.FILE_100K.read_bytes()
+    return file(content, file_name="response_file", content_type="application/x-binary")
+
+
+@app.router.get("/sync-file-response-500K")
+def sync_file_response_500k() -> Response:
+    content = test_data.FILE_500K.read_bytes()
+    return file(content, file_name="response_file", content_type="application/x-binary")
```

Comment:
the standard type for this is `application/octet-stream` is the standard mime type for binary